trajectory_msgs/JointTrajectory joint_trajectory
bool[]  hold
float64 dt
uint32  slot
bool    wipe_all
bool    buffer
bool    run
bool    brake
bool    mode
uint8   run_slot
bool    change_queue
uint8[] new_queue
---
